charts/fluent-bit/values.yaml

```diff
@@ -215,6 +215,12 @@ podLabels: {}
 priorityClassName: ""
 
 env: []
+#  - name: FOO
+#    value: "bar"
+
+envWithTpl: []
+#  - name: FOO_2
+#    value: "bar2"
```

Comment:
```suggestion
envWithTpl: []
#  - name: FOO_2
#    value: "{{ .Values.foo2 }}"
#
# foo2: bar2
```
The example would be better showing the intended use. Maybe also add a comment as to why you'd use it over `env`.